### PR TITLE
Fixed Gruntfile

### PR DESCRIPTION
**Pull request by JSlote**
The syntax of the gruntfile has been repaired (fixing issue https://github.com/dobtco/formbuilder/issues/38 and https://github.com/dobtco/formbuilder/issues/36) and a cleanup task has been added to remove the "compiled" directory when the dist is built.

### CODE REVIEW

**Comment by ajb** [comment]
Thanks for the PR! Let me make some comments inline.

File: Gruntfile.coffee [hunk 1]
```diff
@@ -12,6 +18,7 @@ module.exports = (grunt) ->
   grunt.loadNpmTasks('grunt-contrib-stylus')
   grunt.loadNpmTasks('grunt-contrib-uglify')
   grunt.loadNpmTasks('grunt-contrib-watch')
+  grunt.loadNpmTasks('grunt-contrib-clean')
```
Comment (ajb) [inline Gruntfile.coffee:21]: there is no need for adding these, since the `compiled` directory is in our `.gitignore`.

File: Gruntfile.coffee [hunk 2]
```diff
@@ -29,7 +36,8 @@ module.exports = (grunt) ->
         options:
           namespace: 'Formbuilder.templates'
           processName: (filename) ->
-            filename.replace('<%= srcFolder %>/templates/', '').replace('.html', '')
+            signalStr = "templates/" #strip extra filepath and extensions
```
Comment (ajb) [inline Gruntfile.coffee:39]: Err... what does this do?

**Comment by ajb** [comment]
In addition, thanks for renaming the `.sublimeproject` file... looks like I've been doing it wrong this entire time? But if you could move that out of this PR, that would be great. I'll just commit that directly to master.

File: Gruntfile.coffee (hunk 1 above)
Comment (JSlote) [inline Gruntfile.coffee:21]: I guess I found it useful when developing/modifying the Gruntfile. Older files can get stuck in `compiled` and then find their way into `dist` via concatenation even though Grunt didn't compile them this go-round. It seems right for Grunt to be compiling and concatenating from a clean directory. I guess it's more of a precaution.

File: Gruntfile.coffee (hunk 2 above)
Comment (JSlote) [inline Gruntfile.coffee:39]: The original line wasn't working, so I replaced it with a plain-js version.

File: Gruntfile.coffee (hunk 1 above)
Comment (ajb) [inline Gruntfile.coffee:21]: Got it, that's good enough a reason for me. Thanks for explaining.

**Comment by ajb** [comment]
Thanks again!

**Comment by JSlote** [comment]
Glad to help! I'm busy adding new features and fixing bugs on my fork - I may be pull requesting again here pretty soon.

**Pull request closed without merging**